tests/main-page.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { test, expect } from '@playwright/test'
+
+test('main-page displayed must have the logo image, title, and at least 1 image in the carrousel', async ({
+  page,
+}) => {
+  await page.goto('/')
+  await page.waitForLoadState('networkidle')
+  const logoImage = await page.locator('a[title="logo"] img[alt="logo"]').first()
+  expect(logoImage).toBeTruthy()
+
+  await page.getByTestId('carrousel').click()
```

Comment:
what is this click good for?

Reply:
To know the title of the page it is necessary to click on an element and then take the opportunity to also search the img carousel.if I remove it from error

Reply:
ok but the carrousel does not have a click handler - so strange.. Not sure what you mean by error

Reply:
It doesn't have it but Playwright allows you to click on the element and what it gives you is the text inside to identify the element. We no longer use the text, we use the testId.